doc/source/plugins.rst

```diff
@@ -0,0 +1,82 @@
+.. index:: Plugins
+.. _Plugins:
+
+Plugins
+=======
+
+iocage plugins are a simple and very fast method to get application
+containers installed and configured. At its core, a plugin is a jail
+specifically running one program. Popular programs can be installed
+repeatedly with one line. Additionally, plugins are easily extended by
+users, offering a high level of customizability and functionality.
+
+In structure, a plugin consists of :file:`.json` manifest and
+:file:`.png` icon files.
+
+
+**See what's available**
+
+
+To see a list of all currently available plugins, open a command line
+and type :command:`iocage list -PR` or
+:command:`iocage list --plugins --remote`. The full
+`iocage plugin list <https://raw.githubusercontent.com/freenas/iocage-ix-plugins/master/INDEX>`_
+is also available on GitHub.
+
+Check which plugins are installed on the system with
+:command:`iocage list --plugins` or :command:`iocage list -P`.
+
+
+**Getting started with plugins**
+
+
+.. note:: iocage needs to be `activated <Activate iocage>`_ before
+   plugins can be installed or modified!
+
+
+To get started, open a command line and type
+:command:`iocage fetch --plugins ip4_addr="IF|IP"`. This initial
+:command:`fetch` also supports *dhcp* in the same manner as
+:command:`iocage create`. The ip4_addr needs to conatin the host IP,
+but with the last digits changed. The IP listed for the plugin needs to
```

Comment:
This is not a requirement. What's required is a valid IP address. They could have these on their own subnet, they just need to make sure there's a route and that the address can speak to the world.